pulseapi/entries/models.py

```diff
@@ -64,9 +64,10 @@ def public(self):
                     'help_types',
                     'published_by',
                     'bookmarked_by',
+                    'bookmarked_by__profile__related_user',
                     'published_by__profile',
                     'moderation_state',
-                    'related_creators',
+                    'related_creators__creator__profile__related_user',
```

Comment:
this seems like a pretty big join query. Is this faster than getting the fields on-demand in separate queries?

Reply:
to be profiled/verified on prod